wasm/README.md

```diff
@@ -6,6 +6,30 @@ Code dumped from AssemblyScript with the command line
 yarn build
 ```
 
+<<<<<<< HEAD
```

Comment:
I assume you didn't want to commit that :)

Reply:
oups !!

Reply:
Fixed